pkg/util/httpproxy/upgradeawareproxy.go

```diff
@@ -35,11 +36,13 @@ func NewUpgradeAwareSingleHostReverseProxy(clientConfig *kclient.Config, backend
 	if err != nil {
 		return nil, err
 	}
+	reverseProxy := httputil.NewSingleHostReverseProxy(backendAddr)
+	reverseProxy.FlushInterval = 50 * time.Millisecond
```

Comment:
looks like a super simple fix, which is nice. @smarterclayton @stefwalter is this a reasonable interval? I don't really know what the implications are